tests/v2/test_message_store_queue.nim

```diff
@@ -168,130 +168,6 @@ procSuite "Sorted store queue":
       lastRes.isErr()
       lastRes.error() == "Not found"
 
-  test "Store queue pagination works with predicate - fwd direction":
```

Comment:
Some of these test cases were overlapping with the ones in `test_message_store_queue_pagination`. Removed the duplicated ones, and added the rest of the pagination-related test cases to the other test suite.